treeherder/perf/models.py

```diff
@@ -227,9 +229,50 @@ def _compute_ideal_chunk_size(self, max_timestamp, max_chunk_size):
         return len(older_ids) or max_chunk_size
 
 
-class PerformanceDatum(models.Model):
-    objects = PerformanceDatumManager()
+class Counter(models.Model):
+    """
+    Used for tracking the blocks of ids severed to processes
+    Supports dense, globally unique, ids
+    """
+
+    model = models.CharField(max_length=80, primary_key=True)
+    chunk_size = models.IntegerField(null=False, default=100)
+    count = models.BigIntegerField(null=False)
+
+
+class AutoIncrementMixin:
```

Comment:
Would you mind adding a doc to this class explaining at a high level what it does? Thanks!